metadata.json

```diff
@@ -51,13 +51,14 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-	      "8"
+	      "8",
+	      "9"
```

Comment:
Looks like this contains tabs. Mind fixing it while you're at it? Same below for CentOS